Classes/UT3DmgType_GoliathMachineGun.uc

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2014 GreatEmerald
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
+ *
+ *     (1) Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *
+ *     (2) Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimers in
+ *     the documentation and/or other materials provided with the
+ *     distribution.
+ *
+ *     (3) The name of the author may not be used to
+ *     endorse or promote products derived from this software without
+ *     specific prior written permission.
+ *
+ *     (4) The use, modification and redistribution of this software must
+ *     be made in compliance with the additional terms and restrictions
+ *     provided by the Unreal Tournament 2004 End User License Agreement.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING
+ * IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software is not supported by Atari, S.A., Epic Games, Inc. or any
+ * of such parties' affiliates and subsidiaries.
+ */
+
+class UT3DmgType_GoliathMachineGun extends VehicleDamageType;
+
+defaultproperties
+{
+    bRagdollBullet=true
+    DeathString="%o was punctured by %k's minigun."
+    MaleSuicide="%o turned the minigun on himself."
+    FemaleSuicide="%o turned the minigun on herself."
+    FlashFog=(X=600.00000,Y=0.000000,Z=0.00000)
+    KDamageImpulse=700 //2000 def UT2004, 700 def UT3
+    VehicleDamageScaling=0.6  //0.36 def UT2004, 0.5 def UT3

```

Comment:
If it's 0.5 by default in UT3, then why is it set to 0.6 here?

Reply:
.5 wasn't giving the same amount of damage, 0.6 gives exactly the same damage, no idea why

Reply:
If I recall, I was only getting 8 damage, in UT3 it's 9, with 0.6 I got that

Reply:
That's not good enough. If it's 0.5 in UT3, it must be 0.5 here as well. If it doesn't match what you see, we need to figure out why instead of changing numbers.

Reply:
Open to suggestions because I didn't see anything else

Reply:
I see nothing in the code to suggest that it's 9 damage in UT3.

Reply:
That's what I get in-game on everything but I'll double check when I can, trying to find out about Levi turret throwing shards or not atm

Reply:
I'm getting 9 damage in UT3 on all vehicles on VCTF-Suspense with a single shot

Reply:
Well, put this on 0.5 anyway. There's a number of reasons why there may be a one point difference and `VehicleDamageScaling` is most certainly not it. We can figure it out later.